api/tests/mocha/services/generate-xform.spec.js

```diff
@@ -76,22 +84,64 @@ describe('generate-xform service', () => {
       });
     };
 
-    it('generates form and model', () => runTest('simple'));
+    it('generates form and model', () => runTest('simple', spawned));
 
-    it('replaces multimedia src elements', () => runTest('multimedia'));
+    it('replaces multimedia src elements', () => runTest('multimedia', spawned));
 
-    it('correctly replaces models with nested "</root>" - #5971', () => runTest('nested-root'));
+    it('correctly replaces models with nested "</root>" - #5971', () => runTest('nested-root', spawned));
 
-    it('errors if child process errors', done => {
-      runTest('simple', 'some error')
-        .then(() => done(new Error('expected error to be thrown')))
-        .catch(err => {
-          expect(err.message).to.equal(
-            'Error transforming xml. xsltproc returned code "100", and signal "undefined". ' +
-            'xsltproc stderr output:\nsome error'
-          );
-          done();
-        });
+    it('errors if child process errors', async () => {
```

Comment:
I know these unit test where labeled like this before, but it reads a bit weird, for example: `errors if child process errors`.
Can you please consider updating the unit tests to use `should do .... when ....` labels as we do in other places recently?